### ISSUE TITLE
Spawner break speed configuration is based on client config file

### ISSUE THREAD

**Issue by Razor235T** [mactso/SpawnerInLight#13]
When changing config "spawnerBreakSpeedMultiplier" on the server, it doesn't change it for the client. So if the mod is not installed on the client, speed will be minecraft's default. I personally want to change it to be even slower than mod's default, but all the clients will have to change it manually in their config folder too.

**Comment by mactso** [comment]
That would be correct.  Clients own display of the breaking, the server owns the real breaking.  So even if it shows the cage broke on the client, it won't really be broken until the time passes.   

The Client and Server need to have the same configuration value for it to look right.

In a modpack, the configurations for the clients and server are automatically the same.   

If you have an environment where folks are downloading the mod directly, then they'll need to set it to the value on the server or it will look wrong.   But actual block breaking is done on the server.

Again- it *only* affects the display of the breaking but no the actual breaking/stunning of the spawner.

**Comment by Razor235T** [comment]
No, I can freely set it to 1 (which I guess is vanilla fast) in my own config file and break the spawner fast on the server which has it set to 7. It breaks as it is set to 1 on a server, even no visual lag.
